src/types/guild.ts

```diff
@@ -660,3 +666,40 @@ export interface EditGuildTemplate {
   /** description for the template (0-120 characters) */
   description?: string | null;
 }
+
+/** */
```

Comment:
?

Reply:
The "Document Membership Screening" PR is not yet merged, so the URLs are not available.